### DISCUSSION TITLE
生成式 AI 如何為企業帶來更大的成長優勢

### DISCUSSION THREAD

**Discussion by willychiuchiu** [1101-GS4719-C/Python-Community-GS4719-C#1937] [General]
根據《 2023 產業 AI 化調查》結果，在已使用生成式 AI 工具的企業當中，僅有三成已將資料彙整完成，並準備進行下一個AI專案規劃；另有四成企業則表示持續整理資料。隨著企業在生成式 AI 工具的使用上越來越普及，可以預期比競爭者更擅長或更快速利用工具的企業將會獲得更大優勢；若能以生成式 AI 的使用為基礎，進而推動商業流程變革的企業將會加速成長。

美國財務軟體大廠財捷公司（Intuit）董事長兼共同創辦人Scott Cook等三位作者，在《哈佛商業評論》中將生成式 AI 的運用分為三個層級，分別是使用現有工具提高生產力的第一級；當企業可以用自己的數據對模型進行微調，實現少許的客製化時，層級將會提升到第二級。也就是當企業可以用開源模型，或其他 LLM 公司提供的模型打造客製化的生成式 AI 工具時，並且結合顧客在使用產品或服務的過程中累積的數據和數據，可以使產品更容易與顧客互動，也更容易增加新功能以提高顧客體驗。當企業已能結合使用者回饋，持續改善自己的模型時，就開始朝向第三層級移動，也就是建立自動與連續的數據回饋迴路。當能為公司的產品或服務打造一個獨特的回饋迴路，亦即代表有愈多顧客使用產品時，就會產生愈多回饋訊號，讓生成式 AI 模型持續優化，並帶來更多使用者、使用量及回饋等，逐漸形成一種強大的複合競爭優勢。

參考全文：https://edge.aif.tw/company-gai-strategy/

現今生成式AI在企業的應用已經成為時代趨勢，考驗的是企業要如何妥善的將其應用在自身上，找到對自己營運最有利的方式，這也是這個世代對企業的新課題～

**Comment by benbebobo149** [comment]
對生成式AI感興趣的話，我蠻建議去了解一下RAG是甚麼。因為目前一般公司要跑自己的模型，除了training 會花很多錢之外，持續的running模型所產生的開銷非常龐大。除此之外，就算自己train 出一個模型，其成效不一定會比市面上的LLM如 Gemini，Open AI， Claude還要好用。因此大部分的公司在起始的開發階段，可能會選擇採用現成的模型，以節省時間和成本。等到產品上市到一定規模後，才會開始train自己的模型，來做替換。當然這些東西都是商業機密，大部分公司都會說這是我自家訓練的模型。感謝同學的分享，讓我們能夠共同探索AI的世界。

**Comment by 110101501** [comment]
謝謝同學的分享！
AI在商業的發展上確實是具有優勢，但從中不可忽視的是，發展的同時也是具備風險。以企業的立場而言，若訓練過程無循序漸進、缺乏管理等，也會失敗收場。
因此更凸顯了企業在此領域不斷精進以及優化的重要性。

**Comment by yyujimmy** [comment]
雖然生成式 AI 工具的使用已經普及，但仍有許多企業在資料整合方面存在挑戰。這表明，儘管技術已經成熟，但在實際應用中，企業仍需要面對數據管理和整合等問題。

文章進一步提到了三個層級的生成式 AI 運用，從提高生產力到建立自動與連續的數據回饋迴路。這些層級代表了企業在生成式 AI 應用上的不同階段和發展方向。其中，能夠建立自動回饋迴路的企業將享有更大的競爭優勢，因為他們可以持續改進並優化自己的模型，以滿足客戶需求並提高產品或服務的質量。

**Comment by KH1126** [comment]
生成式 AI 可以生成具有創意和新穎性的內容，這些內容用於創建新產品或服務時，企業可以利用這種技術來探索新的設計和功能，從而在市場上贏得競爭優勢。

**Comment by jeff110303549** [comment]
確實生成式 AI在幾年之內這樣突然衝入大眾的視野，也代表在企業中的應用已經成為一個重要的趨勢。透過生成式 AI 技術，隨著生成式 AI 技術的不斷發展和普及，企業需要找到適合自己的應用方式，並在其中尋求競爭優勢。從提高生產力到實現客製化服務，再到建立自動回饋迴路，企業可以根據自身的需求和發展階段，逐步提升生成式 AI 技術的應用水平，並進一步推動商業流程的變革。
不過我想在成功讓AI企業普及化時，仍需要企業的積極領導和全員的參與，並不斷地優化和調整策略，以確保生成式 AI 的應用能夠真正地為企業帶來價值和競爭優勢。

**Comment by sandyho0307** [comment]
生成式AI在當前的企業營運中，確實已經是一個不可忽視的趨勢。從《2023產業AI化調查》中可看出，越來越多企業意識到整合AI的重要性，尤其是在資料彙整與專案規劃方面的努力顯示出他們為了未來的轉型正在積極準備。這樣的趨勢將會讓那些能有效利用AI工具的企業在競爭中獲得顯著優勢，特別是能夠快速從數據學習和應用洞見到實際操作的公司。
